tap_salesforce/client.py

```diff
@@ -187,9 +190,29 @@ def get_records(
 
         query = f"{select_stm}{from_stm}{where_stm}{order_by_stm}{limit_stm}"
 
-        yield from self._paginate(
-            "GET", f"/services/data/{self._API_VERSION}/queryAll/", params={"q": query}
-        )
+        try:
+            yield from self._paginate(
+                "GET", f"/services/data/{self._API_VERSION}/queryAll/", params={"q": query}
+            )
+        except requests.exceptions.HTTPError as e:
```

Comment:
it needs to be `requests.exceptions.HTTPError`. Tried with `requests.exceptions.Timeout` but, even though the response is due to a timeout, it responds with a 400.